nvme-print-stdout.c

```diff
@@ -2289,10 +2295,16 @@ static void stdout_id_ctrl_ocfs(__le16 ctrl_ocfs)
 {
 	__u16 ocfs = le16_to_cpu(ctrl_ocfs);
 	__u16 rsvd = (ocfs & 0xfffc) >> 2;
+	__u8 copy_fmt_3 = (ocfs >> 3) & 0x1;
+	__u8 copy_fmt_2 = (ocfs >> 2) & 0x1;
 	__u8 copy_fmt_1 = (ocfs >> 1) & 0x1;
 	__u8 copy_fmt_0 = ocfs & 0x1;
 	if (rsvd)
-		printf("  [15:2] : %#x\tReserved\n", rsvd);
+		printf("  [15:4] : %#x\tReserved\n", rsvd);
+	printf("  [3:3] : %#x\tController Copy Format 3h %sSupported\n",
+		copy_fmt_3, copy_fmt_3 ? "" : "Not ");
+	printf("  [2:2] : %#x\tController Copy Format 2h %sSupported\n",
+		copy_fmt_2, copy_fmt_2 ? "" : "Not ");
 	printf("  [1:1] : %#x\tController Copy Format 1h %sSupported\n",
 		copy_fmt_1, copy_fmt_1 ? "" : "Not ");
 	printf("  [0:0] : %#x\tController Copy Format 0h %sSupported\n",
```

Comment:
Since all these bits are treated similarly, would it make more sense to just have a loop over the 4 formats/bits?